ButterCMS/ButterCMSClient.cs

```diff
@@ -664,8 +666,20 @@ public async Task<string> RetrieveContentFieldsJSONAsync(string[] keys, Dictiona
             }
         }
 
+        private string AppendPreviewParams(string queryString)
+        {
+            if (this.preview)
+            {
+                return $"{queryString}&preview=1&test=1";
```

Comment:
Looking good. Last update: please remove `&test=1`. It is a legacy parameter. And also please update README to replace "test" with "preview" in all mentions of it. For example: https://share.getcloudapp.com/9Zum11bZ

Reply:
Fixed, thanks.